.github/workflows/relase.yml

```diff
@@ -0,0 +1,41 @@
+name: Release
+
+on:
+  pull_request_target:
+    branches:
+      - 'release/v[1-9]+.[0-9]+.[0-9]+'
+    types:
+      - closed
+
+jobs:
+  release:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get branch name
+        id: branch-name
+        uses: tj-actions/branch-names@v5.2
+      - uses: actions/checkout@v3
+      - name: Use Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '14.x'
```

Comment:
version 16 would be better, because this is the official LTS now, i prefare: `node:16-alpine`

Reply:
That's right i update it.

The node:16-alpine image is not possible, because of the following error message(this is for runs-on property):
"Value is not accepted. Valid values: "macos-10.15", "macos-11", "macos-12", "macos-latest", "self-hosted", "ubuntu-18.04", "ubuntu-20.04", "ubuntu-latest", "windows-2016", "windows-2019", "windows-2022", "windows-latest".

Reply:
I also update the build pipeline with the version and the limit flag.